<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>subDevComm</class>
 <widget class="QWidget" name="subDevComm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>750</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>500</width>
       <height>750</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#widget{
background-color: rgb(255, 255, 255);
border-style:solid;
border-width:1px;
border-color:rgb(240,240,240);
border-radius:4px;
}
QScrollBar:vertical{ 
    width:8px;  
    border-style:flat;
    border-radius: 4px;
    border:0px;
    background: rgb(255, 255, 255);
} 
QScrollBar::handle:vertical{ 
    background: rgb(230, 230, 230);
    border-radius: 4px;
    width:8px; 
    min-height:50px; 
    border-style:flat;
}
QScrollBar::handle:vertical::hover{ 
    background: rgb(200, 200, 200);
    border-radius: 4px;
    width:8px; 
}
QScrollBar::handle:vertical::pressed{ 
    background: rgb(200, 200, 200);
    border-radius:4px;
    width:8px; 
}
QScrollBar::sub-page:vertical {
     background: rgb(255, 255, 255);
border-style:flat;
}
QScrollBar::add-page:vertical {
     background: rgb(255, 255, 255);
border-style:flat;
}
QScrollBar::add-line:vertical{
     background: rgb(255, 255, 255);
}
QScrollBar::sub-line:vertical {
     background: rgb(255, 255, 255);
}
QScrollBar:horizontal{ 
    height:8px;  
    border-style:flat;
    border-radius: 4px;
    border:0px;
    background: rgb(255, 255, 255);
} 
QScrollBar::handle:horizontal{ 
    background: rgb(230, 230, 230);
    border-radius: 4px;
    height:8px; 
    min-width:50px; 
    border-style:flat;
}
QScrollBar::handle:horizontal::hover{ 
    background: rgb(200, 200, 200);
    border-radius: 4px;
    height:8px; 
}
QScrollBar::handle:horizontal::pressed{ 
    background: rgb(200, 200, 200);
    border-radius:4px;
    height:8px; 
}
QScrollBar::sub-page:horizontal {
     background: rgb(255, 255, 255);
    border-style:flat;
}
QScrollBar::add-page:horizontal {
     background: rgb(255, 255, 255);
    border-style:flat;
}
QScrollBar::sub-line:horizontal {
     background: rgb(255, 255, 255);
}
QScrollBar::add-line:horizontal{
     background: rgb(255, 255, 255);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_windowTitle" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: transparent; </string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0,1">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_windowTitle">
             <property name="toolTip">
              <string/>
             </property>
             <property name="styleSheet">
              <string notr="true">font-weight:bold;
font-size:16px;
color: rgb(50, 50, 50);
</string>
             </property>
             <property name="text">
              <string>子设备交互界面</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>623</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="pushButton_Minimize">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton:hover { 
background-color: rgb(220, 220, 220); 
}
QPushButton { 
image: url(:/png/Minimize.png);
border-width:0;
border-style:outset;
background-color:transparent;
border-radius:2px;
}
</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_Maximize">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton:hover { 
background-color: rgb(220, 220, 220); 
}
QPushButton { 
image: url(:/png/MaximizeOff.png);
border-width:0;
border-style:outset;
background-color:transparent;
border-radius:2px;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_close">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton:hover { 
background-color:rgb(242, 113, 113); 
image: url(:/png/closeW.png)15 0 0 0;
}
QPushButton { 
image: url(:/png/close.png)15 0 0 0;
border-width:0;
border-style:outset;
background-color:transparent;
border-radius:2px;
}
</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>1</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>1</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(200, 200, 200);</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
border-radius:8px;</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>15</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_info" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>70</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>70</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(243, 246, 254, 1);
border-radius:8px;
</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="leftMargin">
             <number>10</number>
            </property>
            <property name="rightMargin">
             <number>10</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QLabel" name="label">
                <property name="styleSheet">
                 <string notr="true">color: rgb(54, 65, 89);
font-size:12px;

</string>
                </property>
                <property name="text">
                 <string>产品号</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_pk">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(54, 65, 89);
font-size:14px;
font-weight:bold;

</string>
                </property>
                <property name="text">
                 <string>- - -</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="styleSheet">
                 <string notr="true">color: rgb(54, 65, 89);
font-size:12px;

</string>
                </property>
                <property name="text">
                 <string>设备号</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_dk">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(54, 65, 89);
font-size:14px;
font-weight:bold;

</string>
                </property>
                <property name="text">
                 <string>- - -</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="styleSheet">
                 <string notr="true">color: rgb(54, 65, 89);
font-size:12px;

</string>
                </property>
                <property name="text">
                 <string>设备状态</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_status">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 85, 0);
font-size:14px;
font-weight:bold;

</string>
                </property>
                <property name="text">
                 <string>离线</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>5</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="pushButton_pass">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">#pushButton_pass{
font-size:16px;
font-weight:bold;
}
QPushButton:pressed{
color: rgb(0, 0, 0);
font-size:16px;
border-radius:0px;
}
QPushButton:checked
{
color: rgb(47, 105, 235);
font-weight:bold;
font-size:16px;
border-radius:0px;
border-bottom-style:solid;
border-bottom-width:2px;
}</string>
                </property>
                <property name="text">
                 <string>透传</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="autoRepeat">
                 <bool>false</bool>
                </property>
                <property name="autoExclusive">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_model">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">#pushButton_model{
font-size:16px;
font-weight:bold;
}
QPushButton:pressed{
color: rgb(0, 0, 0);
font-size:16px;
border-radius:0px;
}
QPushButton:checked
{
color: rgb(47, 105, 235);
font-weight:bold;
font-size:16px;
border-radius:0px;
border-bottom-style:solid;
border-bottom-width:2px;
}
QPushButton:disabled
{
color: rgb(180, 180, 180);
font-weight:bold;
font-size:16px;
background-color: rgb(240, 240, 240);
border-radius:0px;
}</string>
                </property>
                <property name="text">
                 <string>物模型</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="autoExclusive">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QStackedWidget" name="tabWidget">
           <widget class="QWidget" name="tab_sudDev_pass">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QScrollArea" name="scrollArea_pass">
               <property name="widgetResizable">
                <bool>true</bool>
               </property>
               <widget class="QWidget" name="tab_sudDev_passData">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>470</width>
                  <height>499</height>
                 </rect>
                </property>
                <property name="styleSheet">
                 <string notr="true">#tab_sudDev_passData_{
border-style:solid;
border-width:1px;
border-color:rgb(240, 240, 240);
border-radius:4px;
}
</string>
                </property>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_sudDev_module">
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QScrollArea" name="scrollArea_mode">
               <property name="widgetResizable">
                <bool>true</bool>
               </property>
               <widget class="QWidget" name="tab_sudDev_moduleData">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>100</width>
                  <height>30</height>
                 </rect>
                </property>
                <property name="styleSheet">
                 <string notr="true">#tab_sudDev_moduleData_{
border-style:solid;
border-width:1px;
border-color:rgb(240, 240, 240);
border-radius:4px;
}
</string>
                </property>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>1</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>1</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(200, 200, 200);</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_close_2">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>36</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(140,140,140);
font-size:14px;
border-style:solid;
border-width:1px;
border-color:rgb(140,140,140);
border-radius:4px;</string>
          </property>
          <property name="text">
           <string>关闭</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
